tests/foreman/api/test_hostgroup.py

```diff
@@ -148,6 +148,64 @@ def test_bz_1107708(self):
         self.assertEqual(host_attrs['all_puppetclasses'][0]['name'], 'ntp')
 
 
+@skip_if_bug_open('bugzilla', 1235377)
+class MissingAttrTestCase(APITestCase):
+    """Tests to see if the server returns the attributes it should."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create a ``HostGroup`` as ``cls.host_group``."""
+        cls.host_group = entities.HostGroup().create()
+
+    def test_get_content_source(self):
+        """@Test: Read a host group. Inspect the server's response.
+
+        @Assert: The response contains either a ``content_source`` or
+        ``content_source_id`` attribute.
+
+        @Feature: HostGroup
+
+        """
+        wanted_attrs = ('content_source', 'content_source_id')
```

Comment:
I just realized that I can call `read_json()` just once in `setUpClass`, too. Well then. I'm going to update this test. Just a moment, please...